exhibitor-core/src/main/java/com/netflix/exhibitor/core/Exhibitor.java

```diff
@@ -135,7 +136,14 @@ public Exhibitor(ConfigProvider configProvider, Collection<? extends UITab> addi
         processMonitor = new ProcessMonitor(this);
         autoInstanceManagement = new RepeatingActivityImpl(log, activityQueue, QueueGroups.MAIN, new AutomaticInstanceManagement(this), getAutoInstanceManagementPeriod());
 
-        remoteInstanceRequestClient = new RemoteInstanceRequestClientImpl(arguments.remoteConnectionConfiguration);
+        if(arguments.httpsConfiguration.getServerKeystorePath() != null)
```

Comment:
Should we decide here based on the **server** keystore configuration? What if we want TLS connection with peer verification but our server don't expose TLS server?